helm-chart/kuberay-operator/values.yaml

```diff
@@ -45,18 +45,30 @@ readinessProbe:
   periodSeconds: 5
   failureThreshold: 5
 
-rbacEnable: true
-
 batchScheduler:
   enabled: false
 
 # Set up `securityContext` to improve Pod security.
 # See https://github.com/ray-project/kuberay/blob/master/docs/guidance/pod-security.md for further guidance.
 securityContext: {}
 
+
+# If rbacEnable is set to false, no RBAC resources will be created, including the Role for leader election, the Role for Pods and Services, and so on.
+rbacEnable: true
+
+# When crNamespacedRbacEnable is set to true, the KubeRay operator will create a Role for RayCluster preparation (e.g., Pods, Services)
+# and a corresponding RoleBinding for each namespace listed in the "watchNamespace" parameter. Please note that even if crNamespacedRbacEnable
+# is set to false, the Role and RoleBinding for leader election will still be created.
+#
+# Note:
+# (1) This variable is only effective when rbacEnable and singleNamespaceInstall are both set to true.
+# (2) In most cases, it should be set to true, unless you are using a Kubernetes cluster managed by GitOps tools such as ArgoCD.
+crNamespacedRbacEnable: true
```

Comment:
[Minor] I think you might want to rename this field for more clarity. What does "cr" mean: create role?

Reply:
"CR" stands for Custom Resource here. This option is used to control (B) in the PR description. Is `customResourceNamespacedRbacEnabled` a better option?